cpp/cmake/bbp-cmake-format.py

```diff
@@ -132,7 +132,7 @@ def main(**kwargs):
     with build_action_func(args) as action:
         succeeded = True
         for cmake_file in filter_files_outside_time_range(
-            args, collect_files(args.source_dir, args.binary_dir, excludes_re, files_re)
+            args.source_dir, args.applies_on, collect_files(args.source_dir, args.binary_dir, excludes_re, files_re)
```

Comment:
this collect_files here is a rewrite of the function in this file, should be deleted too